### ISSUE TITLE
Got my SafeMoon Tokens then 2 Hours later disappeared from my MetaMask

### ISSUE THREAD

**Issue by WhatsG** [safemoonprotocol/Safemoon.sol#18]
says that it was transfered to a wallet from my metamask to another wallet with no idea how and why.

wallet that took my token: 0xb8303D2ED3e1f4b059DcE9E9dA95ea005B5Dc450

![Untitled](https://user-images.githubusercontent.com/82828790/115264789-783db200-a0eb-11eb-9939-087efcd134c5.png)

**Comment by WhatsG** [comment]
nor did i accept anything that allows my coins to be transferred??

**Comment by WhatsG** [comment]
Another weird thing I noticed is I got 298m coins, 268m coins miracously transfered out and my account is only showing i have 69 coins?? what the heck is going on.....

![Untitled1](https://user-images.githubusercontent.com/82828790/115268962-ae7d3080-a0ef-11eb-97e4-1077b7b6204f.png)

**Comment by rickjunge** [comment]
well, you got ripped by a shitcoin I think. Sorry

**Comment by WhatsG** [comment]
> well, you got ripped by a shitcoin I think. Sorry

That’s it? Happened to me and a handful as im doing my research and people still putting in money into this without knowing why an anomaly like this happens. Are there any support/admins that can look into this?

**Comment by rickjunge** [comment]
No idea. I didn't continue any research after seeing the Source Code and Git Description. It being a bee fork and not beeing updated in the last 2 months (although its not a very new coin) was enough for me, to not buy it.

**Comment by rickjunge** [comment]
Maybe you didn't get ripped, maybe its just a bug. But anyways, I would stay away for now. Better wait for marketplace listing and pay some more. If it's a good coin, it will pay off anyways.

**Comment by theSkele** [comment]
Has your private key been exposed? It was a regular bep20 transfer, nothing special seems to have happened
https://bscscan.com/tx/0x58836a7ead5e2de43de8b98d9a9588e68cc0136e2f087343ad1572f604208b79#eventlog
https://bscscan.com/address/0x8076c74c5e3f5852037f31ff0093eeb8c8add8d3#code

Ok so I was also thinking like did your address ever interact with a smart contract to approve spending of your funds, but the only one you have approved of was SafeMoon... Uhhh sketchy tbh...

**Comment by WhatsG** [comment]
> Has your private key been exposed? It was a regular bep20 transfer, nothing special seems to have happened
> https://bscscan.com/tx/0x58836a7ead5e2de43de8b98d9a9588e68cc0136e2f087343ad1572f604208b79#eventlog
> https://bscscan.com/address/0x8076c74c5e3f5852037f31ff0093eeb8c8add8d3#code
> 
> Ok so I was also thinking like did your address ever interact with a smart contract to approve spending of your funds, but the only one you have approved of was SafeMoon... Uhhh sketchy tbh...

Yup. It is weird. Honestly the only reason I found out was because i checked the day after. I had just gotten it so there was no way i would want to trade or take it out. The amount of time it took me to even understand was crazy. I used metamask and binance. I kept my private keys safe and no one ever touched my laptop. It was weird. Sucks. Probably won't get it back so im staying away. + like i said the balance remaining is not even correct. I have 70 coins left instead of 30m.....

**Comment by arlenner** [comment]
Boss...that's not the address from SAFWMOON website. You probably purchased a lookalike whose only intent was to steal from some of the hype.
